docs: fix info about %include for sample files (#3246)
keptn/lifecycle-toolkit: docs/docs/contribute/docs/code-docs.md

<!-- This is a comment -->
```

## Displaying sample files

Most Keptn configuration is implemented as YAML files
that define a resource
so displaying an example file is very useful.
However, the file content should not be put directly into your doc source.
Instead, the sample file is put into an `assets` directory
and then "included" in your file.
This keeps the documentation source cleaner
and enables us to run tests to ensure that the YAML file is valid
as Keptn evolves.

To implement this:

> 1. Either identify the file you want to include
>    in the `doc/docs/assets/crd/examples` directory
>    or create your sample YAML file in that tree,
>    giving it an appropriate name, including the `.yaml` suffix.

Reviewer: as discussed this morning :)
```suggestion
1. Either identify the file you want to include
   in the `assets` directory next to your documentation file
   or create your sample YAML file in that tree,
   giving it an appropriate name, including the `.yaml` suffix.
```

2. Use the `include <file-path>` shortcode
   to include this file in your documentation source inside a code block.
   For example:

>       ```md

Reviewer: ```suggestion
      ```yaml
```

>       ```yaml

Reviewer: sorry, my bad, this is markdown of course, not yaml
```suggestion
      ```md
```
